applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -327,26 +328,19 @@ void UPwBaseElement<TDim, TNumNodes>::CalculateDampingMatrix(MatrixType&
     const unsigned int N_DOF = this->GetNumberOfDOF();
 
     // Compute Mass Matrix
```

Comment:
Feel free to remove this comment, since it doesn't add anything that the code already tells.

Reply:
Thank you. Indeed the comments just repeat the function names. Removed